lib/DCHttpClient.js

```diff
@@ -0,0 +1,29 @@
+const request = require('request')
+
+class DCHttpClient {
+  constructor () {
+    this.dcAddress = process.env.SGCLOUD_DC_ADDRESS || 'https://dc.shopgate.cloud'
+  }
+
+  login (username, password, cb) {
+    const opts = {
+      method: 'POST',
+      url: `${this.dcAddress}/login`,
+      json: true,
+      body: {username, password}
+    }
+
+    request(opts, (err, res, body) => {
+      if (err) return cb(err)
+      if (res.statusCode !== 200) return cb(new Error(body ? body.message : 'Login failed'))
+      cb(null, body.accessToken)
+    })
+  }
+
+  /*
+    schickt JWT-Token mit an DC (Authorization: Bearer $JWT)
```

Comment:
wrong language and shouldnt this comment be above whatever code is referring to?